CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Added
 - cpcdosc+ outline tree
 - add snippets for compressed files (`decompress/`)
+- add command for download (`download/`)
```

Comment:
As this is not a new keyword but rather a forgotten one, I think it should be moved to the "fixed" section